components/server/src/storage/content-service-client.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { ContentServiceClient } from '@gitpod/content-service/lib/content_grpc_pb';
+import { DeleteUserContentRequest, DeleteUserContentResponse } from "@gitpod/content-service/lib/content_pb";
+import { IDEPluginServiceClient } from '@gitpod/content-service/lib/ideplugin_grpc_pb';
+import { PluginDownloadURLRequest, PluginDownloadURLResponse, PluginHashRequest, PluginHashResponse, PluginUploadURLRequest, PluginUploadURLResponse } from "@gitpod/content-service/lib/ideplugin_pb";
+import { WorkspaceServiceClient } from '@gitpod/content-service/lib/workspace_grpc_pb';
+import { DeleteWorkspaceRequest, DeleteWorkspaceResponse, WorkspaceDownloadURLRequest, WorkspaceDownloadURLResponse } from "@gitpod/content-service/lib/workspace_pb";
+import { inject, injectable } from "inversify";
+import { StorageClient } from "./storage-client";
+
+@injectable()
+export class ContentServiceStorageClient implements StorageClient {
+
+    @inject(ContentServiceClient) private readonly contentServiceClient: ContentServiceClient;
+    @inject(WorkspaceServiceClient) private readonly workspaceServiceClient: WorkspaceServiceClient;
+    @inject(IDEPluginServiceClient) private readonly idePluginServiceClient: IDEPluginServiceClient;
+
+    public async deleteUserContent(ownerId: string): Promise<void> {
+        const request = new DeleteUserContentRequest();
+        request.setOwnerId(ownerId);
+
+        const grcpPromise = new Promise<DeleteUserContentResponse>((resolve, reject) => {
+            this.contentServiceClient.deleteUserContent(request, (err: any, resp: DeleteUserContentResponse) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    resolve(resp);
+                }
+            });
+        });
+        try {
```

Comment:
This is redundant. Just returning or directly `awaiting` the `new Promise<Delete...` is enough